resources/views/Components/FeedbackContainer/FeedbackContainer_component.twig

```diff
@@ -83,9 +95,9 @@
                 {
                     if ( this.feedbackToDelete !== null )
                     {
-                        var feedbackId = this.feedbackToDelete.feedbackId;
-                        var parentFeedbackId = this.feedbackToDelete.parentFeedbackId;
-                        var _self = this;
+                        let feedbackId = this.feedbackToDelete.feedbackId;
```

Comment:
**ACHTUNG**: Über die Scripte diese Plugins läuft kein Babel-Übersetzer, der den Code in ES5 überführt. `let` wird noch nicht von allen Browsern unterstützt. Daher müssen wir hier noch ES5-Syntax verwenden

Reply:
done